### DISCUSSION TITLE
Kafka Static Quota Plugin Not Working

### DISCUSSION THREAD

**Discussion by jslusher** [strimzi/strimzi-kafka-operator#10004] [Q&A]
I'm on version `0.37.0` of the operator running Kafka `3.4.0`. I added these settings in my `Kafka` resource `spec.kafka.config` following [this documentation](https://strimzi.io/docs/operators/in-development/full/deploying.html#proc-setting-broker-limits-str).

```
      client.quota.callback.class: io.strimzi.kafka.quotas.StaticQuotaCallback
      client.quota.callback.static.storage.soft: 150000000000
      client.quota.callback.static.storage.hard: 200000000000
      client.quota.callback.static.storage.check-interval: 5
```

I then tried testing the quota by dumping a large topic into the cluster. The topic took the brokers well over my quota of 200G I had set. Does this version of the operator support the configuration as documented?

**Comment by scholzj** [comment]
I'm not aware of any issues with it (apart from the general design limitations - we are replacing it in #9895).

@im-konge Do you know if there are any STs for the old version of the plugin?
